src/p5.dance.js

```diff
@@ -290,6 +301,12 @@ module.exports = class DanceParty {
     this.allSpritesLoaded = false;
     this.songStartTime_ = 0;
     this.analysisPosition_ = 0;
+    this.world.aiBlockCalled = false;
+    this.world.aiBlockContextUserEventKey = null;
+    // This value is set to `false` if any of the AI blocks in a user program
+    // are called without defined parameters.
+    this.world.aiBlockHasParams = true;
```

Comment:
I agree generally with @breville's comment that double negatives are confusing, but I think its more confusing to have state that is defaulted to something that not really true / is kind of irrelevant to most runs through dance party (ie, when ai blocks aren't used). I'd prefer to default to something like `aiBlockHasInvalidParams = false`

Reply:
This makes senses in this particular case. I'll update! @breville